scripts/test-setup-build/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM --platform=linux/amd64 ubuntu:focal
+
+SHELL ["/bin/bash", "-c"]
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# install tzdata so it doesn't ask the timezone when running
```

Comment:
I'm not super familiar with this code, so I was just wondering what line installs tzdata

Reply:
I wasn't sure myself, so I ran apt rdepends a couple times and found this chain of dependencies leading back to the cmake package:

```root@d6a4ebbae6c8:/# apt rdepends tzdata
tzdata
Reverse Depends:
  Depends: libicu66 (>> 2019c-3ubuntu1~)
root@d6a4ebbae6c8:/# apt rdepends libicu66
libicu66
Reverse Depends:
  Depends: libxml2 (>= 66.1-1~)
root@d6a4ebbae6c8:/# apt rdepends libxml2 
libxml2
Reverse Depends:
  Breaks: zlib1g (<< 2.7.6.dfsg-2)
  Depends: libarchive13 (>= 2.7.4)
  Depends: libcroco3 (>= 2.7.4)
  Depends: gettext (>= 2.9.1)
root@d6a4ebbae6c8:/# apt rdepends libarchive13
libarchive13
Reverse Depends:
  Depends: cmake (>= 3.3.3)
```

Reply:
There may also be other routes from those packages to tzdata as well.